kivy/core/text/__init__.py

```diff
@@ -286,6 +318,15 @@ def __init__(
         self.texture = None
         self.is_shortened = False
         self.resolve_font_name()
+        self._update_padding()
+
+    def _update_padding(self):
+        options = self.options
+        self.options['padding'] = list(self.options['padding'])
+        if options['padding_x']:
+            self.options['padding'][::2] = [options['padding_x']] * 2
```

Comment:
Maybe we can also print a deprecation warning?

Reply:
Ouch! My bad, seen it.

But it's missing the classname that allows the Kivy logger to print it with the proper format.
Left a suggestion on it.